run.go

```diff
@@ -45,12 +45,17 @@ func Run(o Options) (err error) {
 		a.SetProvisioner(astibundler.NewProvisioner(o.Asset))
 	}
 
+	var homeResource string
```

Comment:
Same here, could you name the variable `resourcesPath`?

Reply:
OK